### ISSUE TITLE
Add Multiple People to a meeting at once

### ISSUE THREAD

**Issue by GregoryWillett** [tbpmig/mig-website#232]
It would be super helpful to be able to input a list of uniqnames (copied over from excel) and add all of them to the attendance of a meeting. Especially helpful with first General with no sign-in.

**Comment by mikehand** [comment]
It would seem like this is the only meeting where this would be an issue
yes?

**Comment by mrealm** [comment]
This feature would be a great help for first gen however there are a growing number of people that forget to sign in to events. Having the option to bulk add uniqnames (such as is done to load eligible uniqnames) would be useful.

**Comment by mikehand** [comment]
I agree for the first gen (though by itself that's a low priority issue).
For other events you'd be treating the symptom
